### DISCUSSION TITLE
blog/post/how-we-built-this-blog

### DISCUSSION THREAD

**Discussion by giscus** [The-Zagy/zagy-blog#11] [General]
# blog/post/how-we-built-this-blog

How we made a kendcdodds inspired blog with t3 stack

https://www.zagy.tech/blog/post/how-we-built-this-blog

**Comment by nagy-nabil** [comment]
Yes baby that's how jimmy gets it down

**Comment by dexymore** [comment]
Big dick energy

**Comment by ibrahemaskar11** [comment]
😲😲😲😲😲

**Comment by Mohamed-H-Essa** [comment]
Well said <3

**Comment by alikehel** [comment]
a comment

**Reply by sandstone991** [reply]
that's definietly a comment
